pkg/server/commands/list_objects.go

```diff
@@ -33,7 +33,6 @@ type ListObjectsQuery struct {
 	ListObjectsMaxResults uint32
 	ResolveNodeLimit      uint32
 	ConnectedObjects      func(ctx context.Context, req *ConnectedObjectsRequest, results chan<- string) error
-	CheckResolver         graph.CheckResolver
```

Comment:
this is only needed if the implementation is v0, so i only set it when it's needed